staging/src/k8s.io/component-base/cli/flag/ciphersuites_flag.go

```diff
@@ -109,21 +109,27 @@ func TLSCipherPossibleValues() []string {
 	return cipherKeys.List()
 }
 
-// TLSCipherSuites returns a list of cipher suite IDs from the cipher suite names passed.
-func TLSCipherSuites(cipherNames []string) ([]uint16, error) {
+// TLSCipherSuites returns a list of cipher suite IDs from the cipher suite names passed
+// and returns insecure cipher names matched.
+func TLSCipherSuites(cipherNames []string) ([]uint16, []string, error) {
 	if len(cipherNames) == 0 {
-		return nil, nil
+		return nil, nil, nil
 	}
 	ciphersIntSlice := make([]uint16, 0)
+	insecureCipherNamesMatched := make([]string, 0)
 	possibleCiphers := allCiphers()
 	for _, cipher := range cipherNames {
 		intValue, ok := possibleCiphers[cipher]
 		if !ok {
-			return nil, fmt.Errorf("Cipher suite %s not supported or doesn't exist", cipher)
+			return nil, nil, fmt.Errorf("Cipher suite %s not supported or doesn't exist", cipher)
```

Comment:
```suggestion
			return nil, nil, fmt.Errorf("cipher suite %s not supported or doesn't exist", cipher)
```

see https://github.com/golang/go/wiki/CodeReviewComments#error-strings